test/Microsoft.ML.Predictor.Tests/TestPredictors.cs

```diff
@@ -231,6 +231,18 @@ public void BinaryClassifierLogisticRegressionTest()
             Done();
         }
 
+        [Fact]
+        [TestCategory("Binary")]
+        public void BinaryClassifierSymSgdTest()
+        {
+            //Results sometimes go out of error tolerance on OS X.
```

Comment:
Can we open an issue to enable this test on macOS?

Reply:
We can but linear learners tend to differ on results.

Reply:
At the end of the day, we still need test coverage even if the results differ.  Just because running on a different OS produces different results/behavior doesn't mean we should test on that OS.

Reply:
The other test of SymSGD is enabled on macOS too. I can enable this as well just roughly half the time it will fail because of error tolerance.

Reply:
Just open an issue saying that we should enable this test on macOS.  Then when we work on that issue, we can make it stable as well.

Reply:
Will do.

---
In reply to: [206948457](https://github.com/dotnet/machinelearning/pull/624#discussion_r206948457) [](ancestors = 206948457)